MAUI/TreeMap/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Overview of .NET MAUI TreeMap | Syncfusion
+description: Learn about the introduction of Syncfusion .NET MAUI TreeMap (SfTreeMap) in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Overview of .NET MAUI TreeMap (SfTreeMap)
+
+The Syncfusion .NET MAUI TreeMap control allows you to visually represent hierarchical data with rectangles that are sized and colored based on underlying values.
+
+## Key features 
+ 
+ * Seamlessly bind the TreeMap control to hierarchical data structures.
```

Comment:
might start with heading like "Data Binding" followed by a detailed description of features.

Reply:
Content changed